docs/Concepts/Merge.md

```diff
@@ -0,0 +1,44 @@
+---
+description: What is the Merge?
+---
+
+# The Merge
+
+The Ethereum upgrade known as [The Merge](https://ethereum.org/en/upgrades/merge/) will merge the [Beacon Chain] into
+Ethereum Mainnet, turning Mainnet into a combination of an
+[execution layer and consensus layer](#execution-and-consensus-clients).
+The Merge will transition Mainnet from proof of work to [proof of stake consensus](Proof-of-Stake.md).
+
+## Execution and consensus clients
```

Comment:
Maybe split this into 2 level 3 topics? One focusing on the execution client, and the other on the consensus client.

I find it easier to focus on one concept at a time when reading. Dont judge :)